CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
-cmake_minimum_required(VERSION 3.21)
-set(CMAKE_CXX_STANDARD 20)
+cmake_minimum_required(VERSION 3.30)
```

Comment:
this breaks compilation: https://discourse.cmake.org/t/cmake-3-28-cmake-cxx-compiler-clang-scan-deps-notfound-not-found/9244/3

Reply:
mb

Reply:
do i need to change cxx standard to 20 then?